spec.html

```diff
@@ -12671,14 +12671,14 @@ <h1>
         [[DefineOwnProperty]] (
           _P_: a property key,
           _Desc_: a Property Descriptor,
-        ): either a normal completion containing a Boolean or a throw completion
+        ): a Boolean
```

Comment:
```suggestion
        ): a normal completion containing a Boolean
```

We don't have tooling to enforce it, but we don't allow internal methods to have mixed signatures.